.github/workflows/ci.yml

```
@@ -1,11 +1,28 @@
 name: CI
 on: [push]
 
+env:
+  ZINNIA_VERSION: {{ vars.ZINNIA_VERSION }}
+
```

Is this a variable defined in GitHub repository settings? That makes it difficult to see when we change the Zinnia version.

Please define the Zinnia version inside this workflow file. 

```suggestion
env:
  ZINNIA_VERSION: v0.20.2
```

See how we are doing it in spark-checker:

https://github.com/CheckerNetwork/spark-checker/blob/15cc96d06bce48173459e9874f0d90cb9540e187/.github/workflows/ci.yml#L4-L5

```yml
env:
  ZINNIA_VERSION: v0.20.2
```

When the version is in the workflow file, there will be a pull request to upgrade the version. GitHub will run CI/CD for that change to verify that arweave-checker works with the new Zinnia version.

---

Yes I have defined it in the repository environment variables, but I don't mind having it explicitly defined in code.